src/main.rs

```diff
@@ -84,7 +84,18 @@ fn load_bindings(input: &Path, matches: &ArgMatches) -> Result<Bindings, Error>
         // Load any config specified or search in the input directory
         let mut config = match matches.value_of("config") {
             Some(c) => Config::from_file(c).unwrap(),
-            None => Config::from_root_or_default(input),
+            None => {
+                // In contrast to `Config::from_root_or_default` we look for
+                // cbindgen.toml next to the current _file_. There is always
+                // a parent (at the very least the root), so unwrapping is fine.
```

Comment:
This change looks unrelated now. Can we undo this? Changing the default behavior of config files is a bit subtle, and there's no test for this change.

Reply:
(That said I think it's probably an uncontroversial improvement, I just rather let it land with a separate PR)

Reply:
I reverted it. I can open a separate MR afterwards.